src/stylesheets/elements/_typography.scss

```diff
@@ -68,4 +68,353 @@
 
 .line-height-05 {
   line-height: 0.5rem;
+}
+
+
+
+
+// @mixin h3 {
+//   font-size: units(3);
+//   @include u-text('bold', 'base-dark');
+//   line-height: units(4);
+// }
+
+// @mixin h4 {
+//   font-size: units(2.5);
+//   @include u-text('semibold', 'base-dark');
+//   line-height: units(3);
+// }
+
+// @mixin h5 {
+//   font-size: units(2);
+//   @include u-text('bold', 'base-dark');
+//   line-height: units(2.5);
+// }
+
+// @mixin h6 {
+//   font-size: units(2);
+//   @include u-text('bold', 'base-dark');
+//   line-height: units(2.5);
+// }
+
+
+
+h1, h2, h3, h4, h5, h6 {
+
+  &.heading {
+
+    &--utility-nav {
+      @include u-font('sans', 'lg');
+      @include u-text('primary'); 
+    }
+    
+    &--action-menu {
```

Comment:
Move to sds-text

Reply:
Should this be in an _action.scss instead?